src/imagery/i.fusion.hpf/i.fusion.hpf.md

```diff
@@ -84,11 +84,11 @@ i.colors.enhance b=lsat7_2000_10_hpf g=lsat7_2000_20_hpf r=lsat7_2000_30_hpf
 d.rgb b=lsat7_2000_10_hpf g=lsat7_2000_20_hpf r=lsat7_2000_30_hpf
 ```
 
-[![image-alt](i_fusion_hpf_lsat7_orig_rgb.png)](i_fusion_hpf_lsat7_orig_rgb.png)  
+![image-alt](i_fusion_hpf_lsat7_orig_rgb.png)  
 *Figure: Original Landsat 7 RGB color composite at 28.5m resolution
 (North Carolina, Raleigh subset)*
 
-[![image-alt](i_fusion_hpf_lsat7_hpf_rgb.png)](i_fusion_hpf_lsat7_hpf_rgb.png)  
+[image-alt](i_fusion_hpf_lsat7_hpf_rgb.png)  
```

Comment:
This one doesn't look right, and renders as text in the rich diff view

Reply:
Thanks for catching it!